odata-csdl/5 Schema.md

```diff
@@ -200,7 +200,7 @@ type is the template for an entity: any uniquely identifiable record
 such as a customer or order.
 
 The entity type's name is a [simple identifier](#SimpleIdentifier) that
-MUST be unique within its schema.
+MUST be unique among all direct children of its [schema](#Schema).
```

Comment:
Name uniqueness within a schema sounds to me as if an action name cannot equal the name of a property of an entity type.

I hope this rephrasing makes it clearer.